packages/template-retail-react-app/app/utils/product-utils.js

```diff
@@ -51,40 +51,15 @@ export const getPriceData = (product, opts = {}) => {
     let variantWithLowestPrice
     // grab the variant that has the lowest price (including promotional price)
     if (isMaster) {
-        const variants = product?.variants || []
-        variantWithLowestPrice = variants.reduce(
-            (minVariant, variant) => {
-                const promotions = variant.productPromotions || []
-                const smallestPromotionalPrice = getSmallestValByProperty(
-                    promotions,
-                    'promotionalPrice'
-                )
-                const variantSalePrice =
-                    smallestPromotionalPrice && smallestPromotionalPrice < variant.price
-                        ? smallestPromotionalPrice
-                        : variant.price
-                return variantSalePrice < minVariant.minPrice
-                    ? {minPrice: variantSalePrice, variant}
-                    : minVariant
-            },
-            {minPrice: Infinity, variant: null}
-        )
+        variantWithLowestPrice = findLowestPrice(product)
```

Comment:
with the change you made in the `findLowestPrice`, I don't think we need if/else here anymore. We can just call 
```
currentPrice = findLowestPrice(product)?.minPrice
```

findLowestPrice is already taking account for all types of product type.  🤔

Reply:
I see what you mean.. because we're tracking the `variantWithLowestPrice` though, I think it's better that we leave things as they are now.